directory.py

```diff
@@ -0,0 +1,22 @@
+import directory
+import files
+
+
+class Directory:
+    """
+    Manage directory. Directory can contain Directories or Files.
+    """
+    def __init__(self):
+        self.content = []
+
+    def add_to_directory(self, file: files.File or directory.Directory) -> None:
```

Comment:
Use Union for multiple return types
https://stackoverflow.com/questions/33945261/how-to-specify-multiple-return-types-using-type-hints